Source/EventHorizon/Consumer/Subscription.cs

```diff
@@ -111,6 +117,13 @@ public async Task Start()
             {
                 await _streamProcessor.Start(_cancellationToken).ConfigureAwait(false);
             }
+            catch (Exception ex)
```

Comment:
Does the subscription restart if it fails here? Since the Head won't see this error, it seems like the event horizon will just die and no one will notice.

If that is the case, we should have some retrying around this thing.

Reply:
The ConsumerClient makes sure that the subscription is "resubscribed" if this finished or throws and exception

Reply:
You are right - then we don't need anything here.